xdx.py

```diff
@@ -5,16 +5,16 @@
 from random import randint as randy
 
 def roll_baby_roll(left, size):
-    if left < 1:
-        return 0
-    elif left == 1:
-        return randy(1, size)
-    else:
-        return roll_baby_roll(left-1, size) + randy(1, size)
+    for i in range(left):
+      yield i, randy(1,size)
 
 def main():
     dice = sys.argv[1].split('d')
-    print(roll_baby_roll(int(dice[0]), int(dice[1])))
+    total = 0
+    for dice in roll_baby_roll(int(dice[0]), int(dice[1])):
```

Comment:
It's a bit confusing having dice as vars in both places here

Reply:
https://github.com/masonseeger/xdx-superiority-project/pull/1/commits/da9b75ec2b023a509decc77e27fad60667fac6ad